config/log4j2.yaml

```diff
@@ -0,0 +1,74 @@
+Configuration:
+  status: WARN
+  Appenders:
+    Console:
+      - name: stdout
```

Comment:
[optional] we can merge the appenders as all are same only. Should we add `%t` also in pattern?